lib/grpnames.g

```diff
@@ -25,8 +25,8 @@ NAMES_OF_SMALL_GROUPS :=
   [ "C5 : C4", "C20", "C5 : C4", "D20", "C10 x C2" ], [ "C7 : C3", "C21" ],
   [ "D22", "C22" ], [ "C23" ],
   [ "C3 : C8", "C24", "SL(2,3)", "C3 : Q8", "C4 x S3", "D24",
-      "C2 x (C3 : C4)", "(C6 x C2) : C2", "C12 x C2", "C3 x D8", "C3 x Q8",
-      "S4", "C2 x A4", "C2 x C2 x S3", "C6 x C2 x C2" ], [ "C25", "C5 x C5" ],
+      "C2 x (C3 : C4)", "C3 : D8", "C12 x C2", "C3 x D8", "C3 x Q8", "S4",
+      "C2 x A4", "C2 x C2 x S3", "C6 x C2 x C2" ], [ "C25", "C5 x C5" ],
   [ "D26", "C26" ],
   [ "C27", "C9 x C3", "(C3 x C3) : C3", "C9 : C3", "C3 x C3 x C3" ],
   [ "C7 : C4", "C28", "D28", "C14 x C2" ], [ "C29" ],
```

Comment:
I do not mind doing this, but I am not entirely sure what to do. I am not even sure how to rebase. 

Is the rebase simply to go into my ````SemidirectDecompositions```` branch, and add the command

````rebase gap-system/master````

(after of course fetching updating gap-system/master)?

And if this is done, then how should I add an extra commit where I reformat NAMES_OF_SMALL_GROUPS? Should I somehow go to an earlier commit with some git command, do the change, and commit?